GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -128,6 +129,8 @@ public sealed partial class FormCommit : GitModuleForm
         private readonly TranslationString _commitValidationCaption = new("Commit validation");
 
         private readonly TranslationString _commitMessageSettings = new("Edit commit message templates and settings...");
+        private readonly TranslationString _conventionalCommit = new("Conventional Commits");
+        private readonly TranslationString _conventionalCommitDocumentation = new("Conventional Commits documentation...");
```

Comment:
This string's quite long and makes the context menu mostly empty space. Could it be shorter?

```suggestion
        private readonly TranslationString _conventionalCommitDocumentation = new("About conventional commits...");
```

Or (though doesn't need localisation):

```suggestion
        private readonly TranslationString _conventionalCommitDocumentation = new("conventionalcommits.org");
```

Reply:
I have opted to reduce to only "Documentation..." (as we are in "Conventional Commits" submenu)
The website url is not clear enough for me on what we will find in it...